frontend/packages/dev-console/src/components/projects/details/ProjectDetailsPage.tsx

```diff
@@ -29,6 +29,13 @@ const handleNamespaceChange = (newNamespace: string): void => {
 export const ProjectDetailsPage: React.FC<MonitoringPageProps> = ({ match, ...props }) => {
   const activeNamespace = match.params.ns;
 
+  const canListRoleBindings = useAccessReview({
+    group: RoleBindingModel.apiGroup,
+    resource: RoleBindingModel.plural,
+    verb: 'list',
```

Comment:
What will be the use case if the user has ```list``` permission but do not have ```create``` or ```delete``` permission? 
In this case, can we show this page with all the form field in the read-only state?

see, this user has ```list``` access but not ```create``` and ```delete``` access.
<img width="1105" alt="Screenshot 2020-05-08 at 11 29 05 AM" src="https://user-images.githubusercontent.com/2561818/81375932-23926880-9120-11ea-9479-9bc7d674d46a.png">

cc: @serenamarie125 @christianvogt

Reply:
@vikram-raj please raise an issue to track this. It's a general problem we need to get better at. We see the same missed experience for users with view but not edit access for health checks.

For simplicity right now it's best to stick to hiding if they cannot edit.

Reply:
@christianvogt, For now, do I need to hide the Project Access tab if the user has no permission to create & delete access?

Reply:
We want to hide if the user cannot create role bindings.
There's other checks we _should_ make but better to not let someone even attempt to do something they cannot complete right now.

Reply:
Yes, we should check for ```create``` access instead of ```list``` access here.

Created  an issue to track it https://issues.redhat.com/browse/ODC-3813